src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/CollectionPeriod.cs

```diff
@@ -6,25 +6,34 @@ namespace SFA.DAS.EmployerIncentives.Domain.ValueObjects
 {
     public class CollectionPeriod : ValueObject
     {
-        public CollectionPeriod(byte periodNumber, byte calendarMonth, short calendarYear, DateTime openDate)
+        public CollectionPeriod(byte periodNumber, byte calendarMonth, short calendarYear, DateTime openDate, DateTime censusDate, string academicYear, bool active)
         {
             PeriodNumber = periodNumber;
             CalendarMonth = calendarMonth;
-            CalendarYear =calendarYear;
+            CalendarYear = calendarYear;
             OpenDate = openDate;
+            CensusDate = censusDate;
+            AcademicYear = academicYear;
+            Active = active;
         }
 
         public byte PeriodNumber { get; }
         public byte CalendarMonth { get; }
         public short CalendarYear { get; }
         public DateTime OpenDate { get; }
+        public DateTime CensusDate { get; }
+        public string AcademicYear { get; }
```

Comment:
Any reason the AcademicYear is a string rather than a short?

Reply:
It's not strictly a numeric value, it's a concatenation of two partial year values